lib/reform/form/validation/unique_validator.rb

```diff
@@ -33,7 +38,12 @@ def validate_each(form, attribute, value)
     model = form.model_for_property(attribute)
 
     # search for models with attribute equals to form field value
-    query = model.class.where(attribute => value)
+    if options[:case_sensitive] == false
+      downcased_value = value.nil? ? value : value.downcase
+      query = model.class.where("lower(#{attribute}) = ?", downcased_value)
```

Comment:
Though I don't think you can do SQL Injection with this, it's always best to avoid string interpolation in ActiveRecord queries.